lib/ansible/modules/cloud/vmware/vmware_guest_custom_attributes.py

```diff
@@ -211,6 +245,8 @@ def main():
 
     if vm:
         # virtual machine already exists
+        # TODO: rewrite this part, so that state absent/present
```

Comment:
Should we be adding TODOs or removing them? :)

Reply:
It would be better to fix it in this PR..

Reply:
This will take more code to rewrite, currently this if/else doesn't do much.

Reply:
I just checked the code. Do you need a different output based on the state? I would just add the state to the results dict and remove the if condition entirely.

Reply:
I believe that simply removing this TODO and two ifs will do the trick - it will preserve current behaviour.

But in my opinion it behaves incorrectly based on specified absent/present state - it simply conforms on what been passed from attributes - i.e. if you are passing attributes then they will be presented in VM, but unmentioned attributes will be removed from it (and that might be unexpected).

Should we change the behaviour here?
If state is absent -> remove mentioned attributes and don't touch other linked attributes for VM.

Reply:
@ckotte @jeking3 @Akasurde any ideas here?

Reply:
I think the code needs to do whatever is necessary here to honor the present and absent.  If the VM has 3 attributes (a,b,c) and you issue a module call with state absent for attribute b, then the VM should be left with 2 attributes (a,c).  It looks like the code below just rewrites them no matter what.  Are the deltas managed elsewhere in the code, and at this point it doesn't matter what the state is?